services/skipchain/skipchain.go

```diff
@@ -456,6 +461,20 @@ func (s *Service) lenSkipBlocks() int {
 	return len(s.SkipBlocks)
 }
 
+// saves the actual identity
+func (s *Service) save() {
```

Comment:
Seems this function is exactly the same as in `services/identity/service.go`. Only the file name is changing. Any idea on how it could be reduced to one codebase used twice ? I'm not sure yet since I'm not really too familiar with the identity & skipchains code base